src/containers/Questions/questions.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+
+import propTypes from 'prop-types';
+
+import { Button, message, Input, Progress, Icon, Popconfirm } from 'antd';
+
+import schema from './questionValidation';
+import './questions.css';
+
+const { TextArea } = Input;
+
+const entryData = [
+  {
+    id: 1,
+    heading: 'Today, Iam grateful for',
+    percent: 34,
+  },
+  {
+    id: 2,
+    heading: 'Challenge I face',
+    percent: 68,
+  },
+  {
+    id: 3,
+    heading: 'Iam looking to develop',
+    percent: 100,
+  },
+];
+
+class Questions extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      current: 0,
+      title: '',
+      content: '',
+      errors: {},
+    };
+  }
+
+  confirm = e => {
+    message.success("You didn't make an entry today ): ");
+    const { history } = this.props;
+    history.push('/home');
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value });
+  };
+
+  next = async () => {
+    const { current, title, content } = this.state;
+    try {
+      await schema.validate({ title, content }, { abortEarly: false });
+      return this.setState({
+        current: current + 1,
+        content: '',
+        title: '',
+        errors: {},
+      });
+    } catch (error) {
+      const objError = {};
+      error.inner.forEach(fielderror => {
+        objError[fielderror.path] = fielderror.message;
+      });
+      return this.setState({ errors: objError });
+    }
+  };
+
+  finish = () => {
+    const { title, content } = this.state;
+    const { history } = this.props;
+
+    schema
+      .validate({ title, content }, { abortEarly: false })
+      .then(e => {
+        message.success('Yes, you have added a journal ');
+        history.push('/home');
+      })
+      .catch(error => {
+        const objError = {};
+        error.inner.forEach(fielderror => {
+          objError[fielderror.path] = fielderror.message;
+        });
+        return this.setState({ errors: objError });
+      });
+  };
```

Comment:
Use async await

Reply:
I forgot to change this method as the previous one ,
I will edit it.